README.md

```diff
@@ -1,2 +1,68 @@
-# Ovio.org
-Updated website for [Ovio.org](ovio.org)
+# Ovio website
+
+https://oviohub.com/
+
+A GitHub repo for Ovio static website.
+
+## Development
+
+### Getting started
+
+This project use **[yarn](https://yarnpkg.com/en/)** package manager with **[gatsbyJs](https://www.gatsbyjs.org/)**, a project builder for **[React](https://reactjs.org/)**.
+
+```bash
+yarn
+yarn start
+```
+The website will be available at <http://localhost:8000/>.
+
+### Deployement
+
+To generate the ```public``` folder, run
```

Comment:
In markdown, you can use single ticks for inline (vs. multiline) code formatting (e.g. `public` instead of ```public```)